.github/workflows/phpunit.yml

```diff
@@ -0,0 +1,47 @@
+name: Tests
+on: [pull_request]
+jobs:
+  phpunit:
+    name: PHPUnit
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Cache dependencies
+        uses: actions/cache@v1
+        with:
+          path: ~/.composer/cache/files
+          key: dependencies-composer-${{ hashFiles('composer.json') }}
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.4
+          extensions: dom, curl, libxml, mbstring, zip, pcntl, pdo, sqlite, pdo_sqlite, php-mysql
+          coverage: none
+
+      - name: Copy testing environment file
```

Comment:
Little question: 

Do you copy the `.env` file only for the application key? If yes i think we can simply add it to the `phpunit.xml` file